server/route.go

```diff
@@ -172,6 +172,21 @@ func buildRoutes(app *applicationHandler) http.Handler {
 				})
 			})
 
+			r.Route("/organisations", func(orgRouter chi.Router) {
```

Comment:
@danvixent This should be under the `/ui` router.

Reply:
fixed